resource_dispatcher/tests/integration/test_server_controller.py

```diff
@@ -162,6 +195,11 @@ def test_server_responses(server: HTTPServer, request_data, response_data, resyn
     url = f"http://{server_obj.server_address[0]}:{str(server_obj.server_address[1])}"
     response = requests.post(url, data=json.dumps(request_data))
     result = json.loads(response.text)
+    import logging
+
+    for at in result["attachments"]:
+        logging.error(at)
+
```

Comment:
Question: do we need this ?

Reply:
This is not needed, I've removed it in the commit 6b97a79. Thanks for the catch!